pkg/controller/blockdeviceclaim/blockdeviceclaim_controller.go

```diff
@@ -182,12 +186,16 @@ func (r *ReconcileBlockDeviceClaim) claimDeviceForBlockDeviceClaim(
 		if err != nil {
 			return err
 		}
+		r.recorder.Eventf(selectedDevice,corev1.EventTypeNormal,"BlockDeviceClaimed", "BlockDevice device by %v",instance.Name)
+		r.recorder.Eventf(instance,corev1.EventTypeNormal,"BlockDeviceClaimed","BlockDevice: %v",instance.Spec.BlockDeviceName)
 	}
 
 	err = r.updateClaimStatus(instance.Status.Phase, instance)
 	if err != nil {
 		return err
 	}
+	r.recorder.Eventf(instance,corev1.EventTypeNormal,"BlockDeviceClaimedBound","BlockDeviceClaim is bound")
```

Comment:
@rahulchheda  need to change this to the status we are updating. 
While testing, found out an issue, If the status is pending also, the `BlockDeviceClaimBound` event is generated.